api/crypto/frame_crypto_transformer.cc

```diff
@@ -665,10 +680,11 @@ void FrameCryptorTransformer::decryptFrame(
   }
 
   if (!decryption_success) {
-    if (last_dec_error_ != FrameCryptionState::kDecryptionFailed) {
+    if (last_dec_error_ != FrameCryptionState::kDecryptionFailed || ratchet_count >= key_provider_->options().ratchet_window_size) {
```

Comment:
Will this call multiple type kDecryptionFailed callback?

Reply:
There seems to be no need to judge ratchet_count here, because when decryption_success == false, either ratchet_window_size = 0, or ratchet_count exceeds the number of times，
change to this should be ok
```cpp
  if (!decryption_success) {
      if(key_handler->DecryptionFailure()) {
        if (last_dec_error_ != FrameCryptionState::kDecryptionFailed) {
          last_dec_error_ = FrameCryptionState::kDecryptionFailed;
          onFrameCryptionStateChanged(last_dec_error_);
        }
      }
    return;
  }
```